src-electron/generator/matter/app/zap-templates/templates/app/helper.js

```diff
@@ -828,6 +882,72 @@ function getPythonFieldDefault(type, options) {
   return _getPythonFieldDefault.call(this, type, options);
 }
 
+// Allow-list of enums that we generate as enums, not enum classes.  The goal is
+// to drive this down to 0.
```

Comment:
That is most certainly not the goal for _this_ list.  The goal here is to not touch it.

And I am not sure how you want to deal with the fact that historically this list had entries removed from it....  not sure how far back you want your backwards compat to go.

Reply:
:( . May be just start with this set and if there are any new issues which come up then we re-edit this list again.